### PR TITLE
deps: bump atlas (issue with current version of atlas due to an argum…

### PR DESCRIPTION
**Pull request by masseelch**
…ent change in NewDriver)

Signed-off-by: MasseElch <github@masseelch.de>

### CODE REVIEW

**Review by a8m — APPROVED** [review]

**Review by hedwigz — APPROVED** [review]

**Comment by masseelch** [comment]
@alexsn Sorry to bother you, your initial setup of the CI ran the integration tests on master in every PR. Is there any special reason for that (I am not sure I know why this is done).

**Comment by a8m** [comment]
> @alexsn Sorry to bother you, your initial setup of the CI ran the integration tests on master in every PR. Is there any special reason for that (I am not sure I know why this is done).

The `migration` step was added by me to check that migrating from the master version to the new version is safe and keeps BC (on an existing database). Please, keep it as it prevented a few issues in the past.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -407,13 +405,6 @@ jobs:
...
           restore-keys: |
             ${{ runner.os }}-go-
-      - name: Checkout origin/master
-        run: git checkout origin/master
-      - name: Run integration on origin/master
-        working-directory: entc/integration
-        run: go test -race -count=2 -tags='json1' ./...
-      - name: Checkout previous HEAD
-        run: git checkout -
-      - name: Run integration on HEAD
```
Comment (a8m) [inline .github/workflows/ci.yml:417]: keep it

**Review by a8m — CHANGES_REQUESTED** [review]
Please, revert the CI changes.

**Review by a8m — APPROVED** [review]

**Pull request merged**